### PR TITLE
Javascript3 Week3 Homwork-PROMISESe

### PR DESCRIPTION
**Pull request by Sana-Shabeel**

### CODE REVIEW

**Review by senner007 — COMMENTED** [review]
Hi.

File: javascript/javascript3/week2/main.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  .catch((error) => console.log(error));
+
+//PROMISE THAT RESOLVES AFTER SET TIME
+// 1 create a function with 1 paramter
+// 2 function has to return a promise
+// 3 this will resolve after the seconds have been passed to ii
+// 3 When you have written the promise, use it with async/await
+async function resolvedFn(resolveAfter) {
+  await new Promise((resolve, reject) => {
+    setTimeout(() => resolve(), resolveAfter + "000" - 0); // am turning the arg into a string with three 0s and removing 0 to turn it back into a type number. Uncaught SyntaxError: Octal literals are not allowed in strict mode.
```
[the remark this answers is not in the document]
Comment (Sana-Shabeel) [reply javascript/javascript3/week2/main.js:26]: wouldn't this make it 9000 then ?

[the remark this answers is not in the document]
Comment (Sana-Shabeel) [reply javascript/javascript3/week2/main.js:26]: i wanted to turn it into 8000 with adding extra 1s

[the remark this answers is not in the document]
Comment (Sana-Shabeel) [reply javascript/javascript3/week2/main.js:26]: Ohhh. ok thanks. will correct it

**Pull request merged**